build/containers/archive-analysis/root/etc/systemd/system/archive-import.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=PCP archives import
 Requires=redis.service
-After=redis.service
+After=redis.service stream_maxlen_config.service
```

Comment:
So, perhaps rename this to more like pmseries-config.service  (hyphens seem to be more common in systemd?)